nss-day-cohort-57/tabloidmvc-tactful-tortoises
# Ts createcomment

Changes:
Created CommentRepository.cs
1. GetCommentsByPostId()
2. Add()

Created ICommentRepository.cs
1. with appropriate properties 

Created Comment.cs 
1. with appropriate properties based on the DB

Created CommentCreateViewModel.cs
Created CommentPostViewModel.cs

Created CommentController.cs

To Test:
1. Run code
2. Go to Posts
3. Click on post details
4. click on comment button
5. view all comments, if there are none add a comment
6. ensure that it appears in the posts lists of comments
